.project

```diff
@@ -5,6 +5,11 @@
 	<projects>
 	</projects>
 	<buildSpec>
+		<buildCommand>
+			<name>org.eclipse.buildship.core.gradleprojectbuilder</name>
```

Comment:
Are you one of those that do `git add .` ? 😸

Reply:
Did your update on a different machine, which apparently doesn't have the same git global config, not sure why that sync is broken.